l3kernel/l3tl.dtx

```diff
@@ -891,6 +891,19 @@
 %   \end{texnote}
 % \end{function}
 %
+% \begin{function}[added = 2021-01-23, EXP]{\tl_sort:NN}
+%   \begin{syntax}
+%     \cs{tl_sort:NN} \meta{tl var} \meta{conditional}
+%   \end{syntax}
+%   Like \cs{tl_sort:nN} but this sorts a \meta{tl var}, leaving the result in
+%   the input stream.
+%   \begin{texnote}
+%     The result is returned within \cs{exp_not:n}, which means that the
+%     token list does not expand further when appearing in an
+%     \texttt{x}-type or \texttt{e}-type argument expansion.
+%   \end{texnote}
+% \end{function}
+%
 % \section{The first token from a token list}
```

Comment:
Same comment as for the expandable `\clist_sort:NN`, I think this expandable `\tl_sort:NN` is too likely to get confused with the assignment `\tl_sort:Nn`.